modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptClientCodegen.java

```diff
@@ -144,10 +147,12 @@ public TypeScriptClientCodegen() {
         typeMapping.put("object", "any");
         typeMapping.put("integer", "number");
         typeMapping.put("Map", "any");
+        typeMapping.put("map", "any");
```

Comment:
why do we need this?

Reply:
> Thanks for your contribution! The fix looks good to me - as far as I can see what it does.
> Could you please add unit tests, both to demonstrate the fix working and to prevent regressions in future?
> 
> And if you even feel like going an extra mile here - it would be super beneficial (and highly appreciated! :) ) to add a sample with oneOf/allOf (looks like no existing typescript samples use a spec with this feature - #5335)

Hi 
there is only oneOf but after this fix It would be much easy to add allOf 
I don’t know about unit tests but I'll try to look

Reply:
I added
```typeMapping.put("map", "any"); ```
because I saw it in codegen
``` import { ModelMap } from './ModelMap';```
it was adding to every model 
I researched how it was fixed at typescript-node
and done the same

Reply:
https://github.com/OpenAPITools/openapi-generator/blob/46f8a6733a652544a2914b6bd6dcabf09e49b78c/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractTypeScriptClientCodegen.java#L163-L164

Reply:
@amakhrov ^^